catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/ValidateCommandTest.java

```diff
@@ -240,4 +264,13 @@ public void testQueryWithResults() throws Exception {
     validateCommand.executeWithSubject();
     verify(mockPrinter).printSummary(1, 1);
   }
+
+  private class ValidateCommandUnderTest extends ValidateCommand {
```

Comment:
❓ Is there no better way to inject these? I found this, though not sure it'll work with the Karaf reference annotation. https://github.com/cschneider/osgi-testing-example/blob/master/src/test/java/net/lr/example/testing/PrimeCommandMockTest.java

Reply:
We do it this way in several other locations. I just added it to this as well.